performance-tests/bench/report_parser/main.cpp

```diff
@@ -0,0 +1,50 @@
+#include "ArgumentParser.h"
+#include "TimeSeriesGnuPlotFormatter.h"
+#include "TimeSeriesRawFormatter.h"
+
+int parse_time_series(OutputFormat output_format, Report& report,
+  std::ofstream& output_file_stream)
+{
+  switch (output_format) {
+  case OutputFormat::Gnuplot:
+    TimeSeriesGnuPlotFormatter timeSeriesGnuPlotFormatter;
+    return timeSeriesGnuPlotFormatter.format(report, output_file_stream);
+    break;
+  default:
+    break;
+  }
+
+  TimeSeriesRawFormatter timeSeriesRawFormatter;
+  return timeSeriesRawFormatter.format(report, output_file_stream);
+}
+
+int parse_report(OutputType output_type, OutputFormat output_format,
+  Report& report, std::ofstream& output_file_stream)
+{
+  switch (output_type) {
+  case OutputType::TimeSeries:
+    return parse_time_series(output_format, report, output_file_stream);
+    break;
+  default:
+    break;
+  }
+
+  return EXIT_FAILURE;
+}
+
+int ACE_TMAIN(int argc, ACE_TCHAR* argv[])
```

Comment:
Since this adds a new executable, update the `.gitignore` file (or add one if there isn't one yet).  There is now an error in the Azure Pipelines CI for this.

See the `.gitignore` file in Bench's `test_controller` directory for an example.